worker/src/env.ts

```diff
@@ -144,6 +149,9 @@ const EnvSchema = z.object({
   QUEUE_CONSUMER_POSTHOG_INTEGRATION_QUEUE_IS_ENABLED: z
     .enum(["true", "false"])
     .default("true"),
+  QUEUE_CONSUMER_INGESTION_SECONDARY_QUEUE_IS_ENABLED: z
+    .enum(["true", "false"])
+    .default("true"),
```

Comment:
IMO we can reuse the other variable here. That way we have a single, intuitive kill-switch for ingestion processing. Also, if the first queue stops processing nothing wound end up in the secondary so it's also not super intuitive to have it.